nft/nft_lib/src/main.sw

```diff
@@ -0,0 +1,139 @@
+library nft_lib;
```

Comment:
Libraries dealing with storage as @Braqzen pointed out aren't really well-supported yet. Just making this into a template contract that any can fork, rather than a library that needs to be imported, would be better for now.

Reply:
Agreed, we should make this into a contract rather than a lib.

Reply:
Agree, let's make this a contract rather than a library.